application/src/main/java/net/corda/v5/application/crypto/SigningService.java

```diff
@@ -57,4 +57,44 @@ public interface SigningService {
     @Suspendable
     @NotNull
     Map<PublicKey, PublicKey> findMySigningKeys(@NotNull Set<PublicKey> keys);
+    
+    /**
+     * Deserializes a {@link PublicKey} from a {@code byte} array.
+     *
+     * @param encodedKey The public key represented asa a {@code byte} array.
+     * @return An instance of {@link PublicKey} constructed from the encoded key.
+     */
+    @Suspendable
+    @NotNull
+    PublicKey decodePublicKey(@NotNull byte[] encodedKey);
+
+    /**
+     * Deserializes a {@link PublicKey} from a {@link String}.
+     *
+     * @param encodedKey The public key represented asa a hex encoded {@link String}.
```

Comment:
we refer to this as PEM encoded in the KeyEncodingService which I think is a bit more explanatory?